client/src/containers/user/logic/reducer.ts

```diff
@@ -15,6 +15,21 @@ const routingReducer = createReducer<IUserDataState>(initialState, {
 			accessToken: action.accessToken,
 		};
 	},
+	[actionTypes.UPDATE_USER_NOTIFICATION](state, action: actionTypes.TRequestFailed) {
+		return {
+			...state,
+			requestError: {
+				error: action.error,
+				message: action.message,
+			},
+		};
+	},
+	[actionTypes.CLEAR_USER_NOTIFICATION](state) {
+		return {
+			...state,
+			requestError: undefined,
+		};
+	},
```

Comment:
instead of saving needed info for toast notifications in a state and dispatch `setNotificationState` from controller, you can just put `setNotificationState` from saga